### ISSUE TITLE
Where does `youAreUsingJade` come from on homepage example?

### ISSUE THREAD

**Issue by jordanlev** [jadejs/jade#1901]
This confused me a bit, and hence might confuse other newbies to Jade...

On the example code on your home page, there is a line: `if youAreUsingJade` ... which appears to evaluate to `true` (base don what gets outputted to the html). But this variable is not declared anywhere in the code, which leads me to believe maybe it's a built-in global or something? But I didn't see that anywhere in the docs... so maybe it's not a global but declared elsewhere (outside the scope of the example)?

Thanks!

**Comment by TimothyGu** [comment]
I agree that it might be confusing. It is passed into the template as a local.

**Comment by Laoujin** [comment]
What do you suggest?

- Adding `- var youAreUsingJade = true` to the example
- or adding `jade template.jade -O "{youAreUsingJade : true}"`

**Comment by jordanlev** [comment]
Haven't looked at this in a long time... now that I come back to it, I see that there's also the `pageTitle` variable in there but for some reason I wasn't confused by _that_.

I think showing the variables being passed in would be good, except that the home page is trying to be as simple and basic as possible, so I don't know if adding more stuff to it would actually be a benefit?

I suppose you could set the variables in the template, but that is a somewhat contrived example because usually you pass them in from somewhere else...

Perhaps a link at the bottom that goes to another page on the site with a more full explanation of how to actually run the example (including passing the variables in)?

Not sure... I am confused now about why I was so confused in the first place :)

**Comment by ForbesLindesay** [comment]
I quite like the idea of having something to show the options.  Ideally editable.  Perhaps something like:

```js
output = jade.render(input, {pageTitle: 'Jade', youAreUsingJade: true});
```

We could even make that editable, and just eval it via something like:

```js
var output = Function('input', 'var output;' + src + ';return output;')(input);
```

Which would let people see the impact of editing those variables.  I think that would still be sufficiently minimal and probably would be a lot clearer.
